modules/manage/pages/cluster-maintenance/recovery-mode.adoc

```diff
@@ -0,0 +1,203 @@
+= Recovery Mode
+:description: 
+
+Recovery mode allows you to start Redpanda and form a stable cluster when it fails due to a system crash, OOM errors, or other issues that make a normal startup infeasible. 
+
+In recovery mode, cluster configuration changes and other manual administrative actions are permitted so you can repair and recover the failed cluster. Communication between producers and consumers, as well as topic reads and writes, are disabled.
+
+== About recovery mode
+
+Recovery mode starts a cluster with limited functionality and deactivates normal traffic and data operations for a cluster. This provides a more stable environment for troubleshooting issues and restoring the cluster to a usable state. 
+
+Recovery mode helps to prevent triggering a storm of partition leader elections and replication (such as increased learner replica recovery load after a long broker downtime) that typically occur on startup, especially after some downtime. 
+
+Normal traffic and especially malicious traffic can disrupt availability for all users, including admin users. Recovery mode helps avoid placing the cluster under a heavy load of Kafka API and HTTP Proxy connections as you are working to resuscitate the cluster.
+
+== Requirements
+
+Recovery mode can only be entered during initial cluster formation as brokers start up, and not while they are already running.
+
+== Use recovery mode
+
+To use recovery mode, first configure the node property to enable recovery mode when a broker starts. When enabled, you can also disable partitions, if the cluster failure is localized to just a few partitions.
+
+=== Enable recovery mode
+
+. Run the `rpk redpanda mode` command to set the `recovery_mode_enabled` broker configuration property to `true`.
```

Comment:
Has this been merged yet? It isn't included in our PR to update rpk reference: https://github.com/redpanda-data/docs/pull/104

Reply:
I think it is just `rpk redpanda mode recovery` to set the flag and `rpk redpanda mode production/development` to unset it (depending on whether you want to exit to the production mode or development mode).